src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/service/EditDocsCaseNoteService.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.bailcaseapi.domain.service;
+
+import static uk.gov.hmcts.reform.bailcaseapi.domain.entities.BailCaseFieldDefinition.CASE_NOTES;
+import static uk.gov.hmcts.reform.bailcaseapi.domain.entities.BailCaseFieldDefinition.EDIT_DOCUMENTS_REASON;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.CaseNote;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.field.IdValue;
+import uk.gov.hmcts.reform.bailcaseapi.domain.handlers.postsubmit.editdocs.AuditDetails;
+
+@Service
+public class EditDocsCaseNoteService {
```

Comment:
@Ryan-Nesbitt Can you add a unit test for this class?

Reply:
Marcello added in the extra unit test now